src/persistence.rs

```diff
@@ -515,11 +512,8 @@ mod tests {
             let mut storage =
                 AtomicKeystoreStorage::new(&mut loader, &mut atomic_loader, 1024).unwrap();
             let mut atomic_store = AtomicStore::open(atomic_loader).unwrap();
-            let user_key = UserKeyPair::generate(&mut rng);
-            let ro = random_ro(&mut rng, &user_key);
 
-            let mut updated = stored.clone();
-            updated.txn_state.records.insert(ro, 0, &user_key);
+            let updated = stored.clone();
```

Comment:
This test doesn't really do anything if we don't make a change to `updated`. Can we tweak something in `txn_state` like block height or something?